### PR TITLE
Setup: Initial Jekyll Config (UX-1598)

### PR DESCRIPTION
**Pull request by talbs**
This work adds in basic Jekyll install + site structure

* configuring Jekyll blog settings
* configuring post/element metadata
* creating basic UI architecture
* architecting basic index + element templates
* adding sample posts/elements
* updating README file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,36 @@
+_site
```
Comment (talbs) [inline .gitignore]: NOTE: added some Jekyll-specific files/folders to ignore (namely the compiled/built site)

File: public/pl-static/css/syntax.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
+.highlight .hll { background-color: #ffc; }
```
Comment (talbs) [inline public/pl-static/css/syntax.css]: NOTE: vendor-ish default styling for Pygments (http://jekyllrb.com/docs/templates/#code-snippet-highlighting) for use with our code examples.

File: about.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
```
Comment (talbs) [inline about.md]: NOTE: WIP

File: README.md [hunk 4]
```diff
@@ -1,9 +1,56 @@
 # edX Pattern Library
-This is the working space for the pattern library.
```
Comment (talbs) [inline README.md]: NOTE: Added initial details + basic set up

**Comment by talbs** [comment]
@clrux, mind giving this a once over? A few things to note:

* This is all of the blog infrastructure set-up
* This plans on two main key views - a home/index listing of all elements + a detailed/permalink URL for each element. we can add more views/tweak more complex views as we crush elements in the future
* This contains no patterns
* This contains no styling for the pattern library frame rendering (**coming in a sibling PR very shortly**)
* This contains no local workflow for watching assets (**coming in a sibling PR very shortly**)
* This PR is against ``master`` - I'd like to merge to ``master`` and then have a separate PR when we want to push from ``master`` to the ``gh-pages`` branch which will then update our public domain.
* I've updated the basic README with installation instructions - try it out!

- - -

FYI, @frrrances and @marcotuts

**Comment by clrux** [comment]
:+1:

**Pull request merged**